generator/src/main/java/com/linkedin/pegasus/generator/FileFormatDataSchemaParser.java

```diff
@@ -92,8 +92,9 @@ public FileFormatDataSchemaParser(String resolverPath, DataSchemaResolver schema
               while (entries.hasMoreElements())
               {
                 final JarEntry entry = entries.nextElement();
-                if (!entry.isDirectory() && entry.getName().endsWith(_schemaParserFactory.getLanguageExtension()) &&
-                    (entry.getName().startsWith(_schemaResolver.getSchemasDirectoryName().getName()) || isExtensionEntry(entry)))
+                if (!entry.isDirectory() &&
+                    entry.getName().endsWith(_schemaParserFactory.getLanguageExtension()) &&
+                    (entry.getName().toLowerCase().startsWith(_schemaResolver.getSchemasDirectoryName().name().toLowerCase() + "/")))
```

Comment:
Just use SchemaDirectorName::getName here. Also I agree that we shouldn't ignore case.
Following check should be enough:

entry.getName().startsWith(_schemaResolver.getSchemasDirectoryName().getName() + "/")

Reply:
Ok

Reply:
Addressed